Formula/p/pgcopydb.rb

```diff
@@ -25,12 +25,21 @@ def install
     system "make", "bin"
     libexec.install "src/bin/pgcopydb/pgcopydb"
 
-    (bin/"pgcopydb").write_env_script libexec/"pgcopydb", PATH: "#{Formula["libpq"].opt_bin}:$PATH"
+    (bin/"pgcopydb").write_env_script libexec/"pgcopydb", PATH: "$PATH:#{Formula["libpq"].opt_bin}"
 
     system "make", "-C", "docs", "man"
     man1.install Dir["docs/_build/man/*"]
   end
 
+  def caveats
+    <<~EOS
+      Pgcopydb searches for PostgreSQL command-line tools in your system's PATH environment variable.
+      To use a specific PostgreSQL version's command-line tools, ensure they are accessible in your PATH.
+
+      When no PostgreSQL tools are found in PATH, pgcopydb defaults to using the command-line tools provided by the libpq formula.
```

Comment:
First two lines are general explanation and the last one is the only homebrew specific, IMO.